src/constructorio.js

```diff
@@ -20,7 +20,7 @@ const computePackageVersion = () => {
     versionModifiers.push('domless');
   }
 
-  if (typeof process.env !== 'undefined' && process.env.BUNDLED) {
+  if (typeof process !== 'undefined' && typeof process.env !== 'undefined' && process.env.BUNDLED) {
```

Comment:
You can do this to avoid repetitive "typeof" comparisons

if (process?.env?.BUNDLED) {

}

It has the same behavior as your code, but it's cleaner.